<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size: 10pt;</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <widget class="QTabWidget" name="inputTabWidget">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="documentMode">
           <bool>false</bool>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTabWidget" name="outputTabWidget">
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="documentMode">
           <bool>false</bool>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar {
    background-color: rgb(32, 32, 32);
	border: none;
	padding: 2px;
	spacing: 3px;
}

QToolBar::handle {
    background-color: rgb(60, 60, 60);
    width: 4px;
    border-radius: 2px;
    margin: 4px 2px;
}

QToolBar::handle:vertical {
    height: 4px;
    width: 20px;
    margin: 2px 4px;
}

QToolBar::separator:vertical {
    width: 20px;
    height: 2px;
    margin: 4px 2px;
}

QToolBar::separator {
    background-color: rgb(60, 60, 60);
    width: 2px;
    height: 20px;
    margin: 4px 2px;
    border-radius: 1px;
}

QToolButton {
    background-color: transparent;
    color: rgb(224, 224, 224);
    border: none;
    border-radius: 5px;
    padding: 4px;
    min-width: 24px;
    min-height: 24px;
    margin: 1px;
}

QToolButton:hover {
    background-color: rgb(13, 102, 175);
    border-bottom: 2px solid rgb(0, 0, 0);
	border-right: 2px solid rgb(0, 0, 0);
}

QToolButton:pressed {
    background-color: rgb(16, 125, 220);
    border-bottom: 2px solid rgb(0, 0, 0);
	border-right: 2px solid rgb(0, 0, 0);
}

QToolButton:disabled {
    color: rgb(85, 85, 85);
}

QToolButton QIcon {
    color: rgb(224, 224, 224);
}</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
    background-color: rgb(32, 32, 32);
	padding: 2px;
}

QMenuBar::item {
    padding: 8px 12px;
    margin: 3px 5px;
    border-radius: 5px;
}

QMenuBar::item:selected {
    background-color: rgb(13, 102, 175);
	border-bottom: 2px solid rgb(0, 0, 0);
	border-right: 2px solid rgb(0, 0, 0);
}

QMenuBar::item:pressed {
    background-color: rgb(16, 125, 220);
	border-bottom: 2px solid rgb(0, 0, 0);
	border-right: 2px solid rgb(0, 0, 0);
}

QMenu {
    background-color: rgb(37, 37, 37);
    color: rgb(224, 224, 224);
    border: 1px solid rgb(60,  60,  60);
	border-radius: 5px;
    padding: 4px 0;
}

QMenu::item {
    background-color: transparent;
    padding: 6px 24px 6px 5px;
    margin: 2px 4px;
    border-radius: 5px;
}

QMenu::item:selected {
    background-color: rgb(60, 60, 60);
}

QMenu::separator {
    height: 1px;
    background-color: rgb(60, 60, 60);
    margin: 4px 8px;
}</string>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuText">
    <property name="title">
     <string>Текст</string>
    </property>
    <addaction name="action_15"/>
    <addaction name="action_16"/>
    <addaction name="action_17"/>
    <addaction name="action_18"/>
    <addaction name="action_19"/>
    <addaction name="action_20"/>
    <addaction name="action_21"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Пуск</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="actionLanguage"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuText"/>
   <addaction name="menuRun"/>
   <addaction name="menuHelp"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditUndo"/>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCut"/>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCopy"/>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditPaste"/>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditSelectAll"/>
   </property>
   <property name="text">
    <string>Выделить всё</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>Постановка задачи</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>Грамматика</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>Классификация грамматики</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>Метод анализа</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>Тестовый пример</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>Список литературы</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>Исходный код программы</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpFaq"/>
   </property>
   <property name="text">
    <string>Вызов справки</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
   <property name="toolTip">
    <string>Создать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSaveAs"/>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
   <property name="toolTip">
    <string>Сохранить как</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditRedo"/>
   </property>
   <property name="text">
    <string>Повторить</string>
   </property>
   <property name="toolTip">
    <string>Повторить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
   </property>
   <property name="text">
    <string>Пуск</string>
   </property>
   <property name="toolTip">
    <string>Пуск</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ToolsCheckSpelling"/>
   </property>
   <property name="text">
    <string>Язык</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
